bool blob_detected
int64 blob_x
int64 blob_y
